tests/test_parser.py

```diff
@@ -364,14 +364,16 @@ def test_parser_id_1074():
            'alias': 'Sample'
            }
     record = parser.parse_id_1074(d, rec)  # System    -   Shutdown Initiated
-    data = d["Event"]["EventData"]["Data"]
-    assert(record["account"]) == "SYSTEM"
-    assert(record["details"]) == "Cause: restart"
-    assert(record["computer_name"]) == "LAPTOP-9KUQNI2Q"
-    if data[3]["#text"] == "0x500ff":
-        assert(record["description"]) == "Shutdown Initiated"
-    else:
-        assert(record["description"]) == "ERROR"
+    assert (record) == None  # This is the case because Bruce "tosses" any 1074 records where Data param4 != 0x500ff.
```

Comment:
Can you change this test to cover both of the cases for when we parse a record with id 1074 that we do want and when we parse one that we toss?

Reply:
@zjmonroe23